jsk_perception/launch/darknet_yolo_object_segmentation.launch

```diff
@@ -0,0 +1,12 @@
+<launch>
+  <arg name="input_image" default="/camera/rgb/image_rect_color" />
+
+  <node name="darknet_yolo_object_detection"
+        pkg="jsk_perception" type="darknet_object_detection"
+        output="screen">
+    <param name="network_weight_path" value="$(find darknet)/weights/yolo.weights" />
+    <param name="network_config_path" value="$(find darknet)/cfg/yolo.cfg" />
+    <param name="label_path" value="$(find darknet)/data/coco.names" />
```

Comment:
@k-okada Could you try with the latest revison and check this problem still occurs? In my environment this error does not show up.
And, what do you mean `other than add label directory under ~/.ros/data? `